frontend/src/components/marketDetails/MarketDetailsLayout.jsx

```diff
@@ -1,75 +1,105 @@
+        </h1>
+        <div className='flex items-center space-x-2 text-sm text-gray-400'>
+          <Link
+            to={`/user/${market.creatorUsername}`}
+            className='hover:text-blue-400 transition-colors duration-200'
+          >
+            <span role='img' aria-label='Creator'>
+              {creator.personalEmoji}
+            </span>
+            @{market.creatorUsername}
+          </Link>
+          <span>•</span>
+          <span>🪙 {currentProbability.toFixed(2)}</span>
+        </div>
+      </div>
+
+      <div className='mb-4'>
+        <MarketChart
+          data={probabilityChanges}
+          currentProbability={currentProbability}
+          title='Probability Changes'
+          className='w-full'
+        />
+      </div>
+
+      <div className='mb-4'>
+        <button
+          onClick={() => setShowFullDescription(!showFullDescription)}
+          className='w-full py-2 bg-gray-700 hover:bg-gray-600 transition-colors duration-200 rounded-lg text-center text-sm'
+        >
+          {showFullDescription ? 'Hide Description' : 'Show Full Description'}
+        </button>
+      </div>
 
-function MarketDetailsTable({ market, creator, numUsers, totalVolume, currentProbability, probabilityChanges }) {
-    const [showFullDescription, setShowFullDescription] = useState(false);
+      <div className='mb-4 bg-gray-800 p-4 rounded-lg'>
+        <p
+          className={`text-sm break-words whitespace-pre-wrap ${
+            showFullDescription
+              ? ''
+              : 'sm:max-h-24 h-16 overflow-y-auto sm:overflow-hidden'
+          }`}
+          style={{
+            wordBreak: 'break-word',
+            overflowWrap: 'break-word',
+            hyphens: 'auto',
+          }}
+        >
+          {market.description}
+        </p>
+      </div>
 
-    // Function to truncate the description to 50 characters
-    const truncateDescription = (description) => {
-        if (description.length > 50) {
-            return `${description.substring(0, 100)}...`;
-        }
-        return description;
-    };
+      <div className='grid grid-cols-2 sm:grid-cols-4 gap-2 text-center mb-4'>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Users</div>
+          <div className='text-sm font-semibold'>👤 {numUsers}</div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Volume</div>
+          <div className='text-sm font-semibold'>
+            📊 {totalVolume.toFixed(2)}
+          </div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Comments</div>
+          <div className='text-sm font-semibold'>💬 0</div>
+        </div>
+        <div className='bg-gray-800 p-2 rounded-lg'>
+          <div className='text-xs text-gray-400'>Closes</div>
+          <div className='text-sm font-semibold'>
+            📅{' '}
```

Comment:
Another comment, this applies to the case where a market has been RESOLVED as either YES or NO.

So if you hit the API endpoint for a particular market that has been resolved NO, you will get:

```
<div class="toolbar"><div class="devtools-separator"></div><div class="devtools-searchbox"><input class="searchBox devtools-filterinput" placeholder="Filter JSON" value=""></div></div><div class="panelContent" id="json-scrolling-panel" tabindex="0">
  |  
-- | --
market |  
id | 4
questionTitle | "Test Create"
description | "Test!"
outcomeType | "BINARY"
resolutionDateTime | "2024-08-16T09:59:00Z"
finalResolutionDateTime | "2024-08-15T21:40:59.414181Z"
utcOffset | 300
isResolved | true
resolutionResult | "NO"
initialProbability | 0.5
creatorUsername | "patrick"
createdAt | "2024-08-15T21:40:18.610412Z"
creator |  
username | "patrick"
displayname | "Patrick"
usertype | "REGULAR"
initialAccountBalance | 0
accountBalance | -41
personalEmoji | "😃"
description | "Stuff"
personalink1 | "https://www.google.com/"
probabilityChanges |  
0 |  
probability | 0.5
timestamp | "2024-08-15T21:40:18.610412Z"
numUsers | 0
totalVolume | 0

</div>
```

Note that `isResolved` is `TRUE` and the `resolutionResult` status is `NO`

However, if you look at a market that has not been resolved yet, even if the close date has passed, you get:

```
isResolved	false
resolutionResult	""
```
So basically, in both the `MarketDetailsLayout.jsx` and the `MarketTables.jsx` we should have additional logic to display whether the market has been resolved or not and the resolution direction, either YES or NO.

Reply:
Fixed here https://github.com/openpredictionmarkets/socialpredict/pull/242/commits/5a5d2b7b3e13049d3bc714ecab94fa8caaa5a754